lifx-core/src/lib.rs

```diff
@@ -2577,7 +2607,7 @@ mod tests {
             protocol: 1024,
             source: 1234567,
         };
-        frame.validate();
+        assert!(frame.validate().is_ok());
```

Comment:
```suggestion
        frame.validate().unwrap();
```

In tests I prefer to `unwrap`, so the output of a failed test will include the error value.

Reply:
gotcha